cmd/vicadmin/logs.go

```diff
@@ -378,14 +378,15 @@ func findSeekPos(f *os.File) int64 {
 		// Go back nBytes from the last newline we've seen (or beginning-of-file),
 		// and read the next nBytes
 		readPos := lastnl - int64(len(buf))
-		if pos < 0 {
+		// Fix here.  Previously was "pos < 0".
+		if readPos < 0 {
 			readPos = 0
 		}
 		bufend, err := f.ReadAt(buf, readPos)
```

Comment:
bufend isn't the end of the buffer, it's the number of bytes read. The index in the buffer should probably be `readPos + bufend`